DEVELOPER_GUIDE.md

```diff
@@ -140,9 +148,5 @@ Run from `bundle-workflow` before making pull requests.
 ```
 cd bundle-workflow
 
-git status -s | grep -e "[MA?]\s.*.py" | cut -c4- | xargs pipenv run black
```

Comment:
The pre-commit hook will run the checks, but doesn't auto-fix anything (and shouldn't). Let's put back the cheatsheet to fix things: maybe say "The pre-commit hook checks for ..." and "Auto-fix format and sort imports by running  `... run black`, and `isort .`"?